functions/src/messages/SendMessageHandler.ts

```diff
@@ -0,0 +1,141 @@
+import * as functions from 'firebase-functions';
+import { TWILIO_SENDER_PHONE, TWILIO_SID, TWILIO_TOKEN } from '../config';
+import { Entity, User, Recipient } from '../../../shared/src/types';
+import { FirestoreAdmin } from '../../../shared/src/firebase/FirestoreAdmin';
+import { sendSms } from '../../../shared/src/utils/messaging/sms';
+import { renderTemplate } from '../../../shared/src/utils/templates';
+
+export interface SendMessagesFunctionProps {
+	communicationChannel: 'sms' | 'email';
```

Comment:
👍 nicely typed!
nit: alternatively, we could use an enum, then we don't need to do the string comparisons afterwards when using it.